docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  caldera:
+    image: python:3
+
+    expose:
+    - "8880"
+    - "8888"
+
+    ports:
+      - "8880:8880"
+      - "8888:8888"
+    working_dir: /usr/src/app
+
+    volumes:
+      - ./:/usr/src/app
+
+    command: /bin/bash -c "pip install --no-cache-dir -r requirements.txt && python server.py -E local"
```

Comment:
the -E local can be removed, I believe

Reply:
Correct. I just pushed two edits to remove the `-E local` and `port 8880` port binding